src/engine/strat_engine/backstore/metadata/bda.rs

```diff
@@ -156,6 +156,12 @@ impl BDA {
             None => return Ok(None),
         };
 
+        // Assume that, since a valid StaticHeader was found on the device,
+        // that this implies that BDA::initialize() was succesfully executed
+        // sometime in the past. Since that is the case, valid MDA headers
+        // were written to the device. If there is an error loading the
```

Comment:
The last sentence is a bit complicated to read. What about `Returns an error if there is an error when loading the MDARegions, which can only be caused by an I/O error or invalid MDA headers.`?

Reply:
Ok, I'll go w/ your version.